lab-adam/assets/one.txt

```diff
@@ -0,0 +1,141 @@
+
+Vestibulum ullamcorper lacus justo, sit amet vehicula quam commodo et. Fusce eu mauris hendrerit dui aliquam pharetra. Nam interdum semper odio vel malesuada. Aliquam nec semper leo. In pharetra metus at purus maximus, ac volutpat justo vestibulum. Nam faucibus rhoncus luctus. Donec ipsum nisl, pulvinar a libero ac, tempor imperdiet lacus. Maecenas eget aliquam odio, ut cursus purus. Phasellus placerat sed lorem ac rutrum. In laoreet pretium congue. Suspendisse viverra vitae lectus finibus tristique. Nulla maximus hendrerit metus quis facilisis. Curabitur dui risus, gravida eget dignissim id, iaculis sed elit. Nulla feugiat neque ligula, aliquam ornare felis tempus eu. Praesent lobortis ultricies lectus, nec vehicula massa malesuada sit amet.
+
+Donec malesuada neque at finibus lobortis. Sed ultricies, lectus a aliquet sollicitudin, leo dui iaculis neque, at pharetra ex mauris et dui. Donec vestibulum luctus tristique. In pharetra neque ac augue ultricies porttitor. In sagittis vulputate dignissim. Donec ultrices augue semper, imperdiet augue vel, venenatis urna. Aenean pretium ornare quam id ultrices.
+
+Proin vel fringilla urna. Nunc hendrerit arcu massa, ut lobortis velit euismod et. Nam elit elit, tincidunt a egestas a, imperdiet vel tellus. Morbi rutrum tristique lorem, a euismod nunc viverra vitae. Curabitur non arcu in nulla convallis euismod. Aliquam gravida, orci bibendum pharetra scelerisque, sapien sapien pellentesque metus, a tempus leo ex id lorem. Nulla volutpat bibendum ligula ac ornare. Vestibulum id magna ut sapien tincidunt commodo sit amet nec ante.
+
+Aenean et tortor suscipit, aliquet tortor vel, luctus justo. Aenean tincidunt id ante ut auctor. Nam molestie leo sed mauris bibendum pellentesque. Vivamus rutrum urna sed mi finibus consectetur. Nullam at consequat ipsum. Aliquam sed felis turpis. Morbi efficitur lobortis urna, et pellentesque tortor consequat ut. Sed convallis libero a arcu euismod cursus. Aliquam consectetur hendrerit nisl, id placerat nunc tincidunt eget. Morbi placerat rutrum erat, et venenatis libero. Ut scelerisque enim non congue pretium. Proin congue condimentum leo, sit amet elementum magna rutrum a. Nunc ac arcu ac augue sagittis cursus ac vitae nisl.
```

Comment:
Good work trying to make sure that your files are being read in the correct order even when the first file is much larger.